services/workflows-service/src/collection-flow/collection-flow.service.ts

```diff
@@ -287,13 +287,18 @@ export class CollectionFlowService {
     );
   }
 
-  async getCollectionFlowContext(tokenScope: ITokenScope): Promise<DefaultContextSchema> {
+  async getCollectionFlowContext(
```

Comment:
you can cover the regression we had by testing the endpoint fo this function, making sure it return config, on the kyb app side, you can just unit test the override logic works I guess

Reply:
done